semantic_segmentation/scripts/train.py

```diff
@@ -20,16 +20,15 @@
 from dataset import Dataset
 
 
-class UNET_LT(pl.LightningModule):
-    def __init__(self, lr, **kwargs):
+class Suad_Semseg(pl.LightningModule):
+    def __init__(self, arch, lr, num_classes, **kwargs):
         super().__init__()
-        self.Net = U2NET_lite()
-        self.Net.cuda(1)
+        self.net = arch()
```

Comment:
after a while - it will be better to pass the object instance named as `net`:

```
self.net = net
```

more intuitive